### PR TITLE
Refactor LoadEarlier component to FC

### PR DESCRIPTION
**Pull request by danielmejiadev**
@xcarpentier Could you please review this changes?
Rewrite the LoadEarlier component in order to migrate to FC

Thanks, Have a nice day.

### CODE REVIEW

File: src/LoadEarlier.tsx [hunk 1]
```diff
@@ -55,68 +56,66 @@ export interface LoadEarlierProps {
...
-    wrapperStyle: StylePropType,
-    textStyle: StylePropType,
-    activityIndicatorStyle: StylePropType,
-    activityIndicatorColor: PropTypes.string,
-    activityIndicatorSize: PropTypes.string,
-  }
-
-  renderLoading() {
-    if (this.props.isLoadingEarlier === false) {
+  const renderLoading = useCallbackOne(() => {
```
Comment (xcarpentier) [inline src/LoadEarlier.tsx:61]: Please move this code directly into return jsx even if it was into a render function member of the class, as I see there is no render props right?

Comment (danielmejiadev) [reply src/LoadEarlier.tsx]: Yeah, I got
Comment, solved

**Comment by danielmejiadev** [comment]
@xcarpentier Comments solved

File: src/LoadEarlier.tsx [hunk 2]
```diff
@@ -55,68 +55,63 @@ export interface LoadEarlierProps {
...
-        </Text>
-        <ActivityIndicator
-          color={this.props.activityIndicatorColor!}
-          size={this.props.activityIndicatorSize!}
-          style={[styles.activityIndicator, this.props.activityIndicatorStyle]}
-        />
+  return (
+    <TouchableOpacity
+      style={[styles.container, props.containerStyle]}
+      onPress={() => onLoadEarlier?.()}
```
Comment (xcarpentier) [inline src/LoadEarlier.tsx:64]: It should be `onPress={onLoadEarlier}`

Comment (xcarpentier) [inline src/LoadEarlier.tsx:64]: It should be `onPress={onLoadEarlier}`

Comment (danielmejiadev) [reply src/LoadEarlier.tsx]: Solved

**Comment by danielmejiadev** [comment]
@xcarpentier Comments solved

**Pull request merged**